src/api/models/bedrock.py

```diff
@@ -191,6 +191,19 @@ class BedrockModel(BaseChatModel):
             "tool_call": True,
             "stream_tool_call": False,
         },
+        # claude 3.5 cross-region inference profile
+        "us.anthropic.claude-3-5-sonnet-20240620-v1:0": {
+            "system": True,
+            "multimodal": True,
+            "tool_call": True,
+            "stream_tool_call": True,
+        },
+        "us.anthropic.claude-3-5-sonnet-20241022-v2:0": {
```

Comment:
Which documentation did you use for this profile ID? I couldn't locate it on https://docs.aws.amazon.com/bedrock/latest/userguide/cross-region-inference-support.html.

Reply:
Got it